pipx/Venv.py

```diff
@@ -176,7 +188,44 @@ def get_venv_metadata_for_package(self, package: str) -> PipxVenvMetadata:
             data["app_paths_of_dependencies"][dep] = paths
             data["apps_of_dependencies"] += [path.name for path in paths]
 
-        return PipxVenvMetadata(**data)
+        return VenvMetadata(**data)
+
+    def update_package_metadata(
+        self,
+        package: str,
+        package_or_url: str,
+        pip_args: List[str],
+        include_dependencies: bool,
+        include_apps: bool,
+        is_main: bool,
```

Comment:
nit: recommend using `is_main_package` rather than `is_main` since I wasn't sure if this was referring to the `main` function or the main package for the venv.

Reply:
Done.